examples/eks_cluster_webshop/lb.tf

```diff
@@ -0,0 +1,9 @@
+module "load_balancer_controller" {
+  source = "git::https://github.com/DNXLabs/terraform-aws-eks-lb-controller.git"
```

Comment:
How did you choose this module? Doesn't look very popular one.

Reply:
reached out to Marcin as well, but seems like a fairly simple module and there isn't any more popular alternatives

wonder if we could replace source with
https://registry.terraform.io/modules/DNXLabs/eks-lb-controller/aws/latest though?

Reply:
I didn't find anything better than this one. Writing it on my own would be copy-paste of this one.
 Source for module updated.